homework.md

```diff
@@ -92,13 +91,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send your changes to the stage 
```

Comment:
`git add` stages the changes, `git commit` saves your changes to your local repo